src/components/Tile/Tile.css

```diff
@@ -0,0 +1,11 @@
+.tile {
+  height: 70px;
+  width: 70px;
+  margin: 5px;
+  background-color: white;
+  border-radius: 50%;
+  border: 5px solid navy;
+  /* cursor: pointer; */
+  /* cursor: url(redDisc); */
+  /* cursor: url("../../images/red_disc.svg"); */
```

Comment:
Don't leave commented out code in your files.

Reply:
I'll get rid of all the uncommented code